src/model_config_tests/qa/test_access_esm1p5_config.py

```diff
@@ -0,0 +1,167 @@
+# Copyright 2024 ACCESS-NRI and contributors. See the top-level COPYRIGHT file for details.
+# SPDX-License-Identifier: Apache-2.0
+
+"""ACCESS-ESM1.5 specific configuration tests"""
+
+import re
+import warnings
+from typing import Any
+
+import pytest
+
+from model_config_tests.util import get_git_branch_name
+
+### Bunch of expected values for tests ###
+VALID_REALMS: set[str] = {"atmosphere", "land", "ocean", "ocnBgchm", "seaIce"}
+VALID_KEYWORDS: set[str] = {"global", "access-esm1.5"}
+VALID_NOMINAL_RESOLUTION: str = "100 km"
+VALID_REFERENCE: str = "https://doi.org/10.1071/ES19035"
+VALID_PREINDUSTRIAL_START: dict[str, int] = {"year": 101, "month": 1, "day": 1}
+VALID_HISTORICAL_START: dict[str, int] = {"year": 1850, "month": 1, "day": 1}
+VALID_RUNTIME: dict[str, int] = {"year": 1, "month": 0, "day": 0}
+VALID_RESTART_FREQ: str = "20YS"
```

Comment:
@blimlim talked to Martin and settled on "10YS" as a standard for these configs. Sorry for the bum steer on that one.

Reply:
Updated in https://github.com/ACCESS-NRI/model-config-tests/pull/41/commits/c4d8a7d130b1a3764c34ba3c7cd0cbd7cb7971f8